internal/provider/resource_hvn_route.go

```diff
@@ -67,16 +65,18 @@ func resourceHvnRoute() *schema.Resource {
 				Required:    true,
 				ForceNew:    true,
 			},
-			// Optional inputs
+			// Computed outputs
 			"project_id": {
-				Description:  "The ID of the HCP project where the HVN route is located.",
-				Type:         schema.TypeString,
-				Optional:     true,
-				ForceNew:     true,
-				ValidateFunc: validation.IsUUID,
-				Computed:     true,
+				Description: "The ID of the HCP project where the HVN route is located. Always matches the project ID in `hvn_link`. Setting this attribute is deprecated, but it will remain usable in read-only form.",
```

Comment:
Out of curiosity, when `computed` can this field be set in the resource even though when overwritten?